epifor/data/csse.py

```diff
@@ -23,27 +24,61 @@
 class CSSEData:
     def __init__(self):
         self.df = None
-
-    def load(self, pattern):
+        self.HIST_FILE_NAME = "csse_history_data.h5"
+
+    @staticmethod
+    def convert_date(string):
+        " Convert string from csv to pandas allowed format"
+        try:
+            return datetime.datetime.strptime(string, "%m/%d/%Y").strftime("%Y%m%d")
+        except ValueError:
+            return datetime.datetime.strptime(string, "%m/%d/%y").strftime("%Y%m%d")
+
+    @staticmethod
+    def nearest_date(items, pivot):
+        " Find the nearest date to start_date from config "
+        items = [datetime.datetime.strptime(x, "%Y%m%d") for x in items]
+        pivot = datetime.datetime.strptime(pivot, "%Y%m%d")
+        return min(items, key=lambda x: abs(x - pivot)).strftime("%Y%m%d")
+
+    def load(self, pattern, by_date):
         dfs = []
-        for name in ["Confirmed", "Deaths", "Recovered"]:
-            df = pd.read_csv(pattern.format(name.lower()), header=0)
+        shortest_list_dcs = None
+        by_date = by_date.strftime("%Y%m%d")
+        for name in ["confirmed", "deaths", "recovered"]:
+            df = pd.read_csv(pattern.format(name), header=0)
+
             dcs = list(df.columns)[4:]
+            # Rename columns to format pandas would accept
```

Comment:
btw. pandas accepts almost any format, there are things like `pd.to_datetime` and you could also pass formatter when reading CSV. But if this works, it's fine